sass/_colors.scss

```diff
@@ -47,3 +47,6 @@ $data-series-colors: #7fc675, #2e7e9b, #87768d, #a49f7f, #7ac9a4, #c1cb89, #e3a2
 $flash-success-background: rgba($success, 0.9);
 $flash-alert-background: rgba($alert, 0.9);
 $flash-error-background: rgba(lighten($error, 5%), 0.9);
+
+// Table colors
+$light-smoke: #f2f2f2;
```

Comment:
I would consider using the existing variable `$grey-lightest` and seeing if that looks okay, as it is a very similar colour. It's always nice to minimise how many colours we use.